IMA/FairPlayIMAPlayer/objc/FairPlayIMAPlayer/ViewController.m

```diff
@@ -103,6 +103,69 @@ - (void)setup
     // Get the shared SDK manager
     BCOVPlayerSDKManager *sdkManager = [BCOVPlayerSDKManager sharedManager];
 
+    // Create an authorization proxy for FairPlay
+    // using the FairPlay Application ID and the FairPlay Publisher ID
+    BCOVFPSBrightcoveAuthProxy *proxy = [[BCOVFPSBrightcoveAuthProxy alloc] initWithPublisherId:kFairPlayPublisherId
```

Comment:
this is optional and is not required for Dynamic Delivery

Reply:
@sahlas do you mean it needs a comment stating that it's optional for DyD?